aws-attach-roles/iam-kubecost-savings-policy.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
@jessegoodier I see this is a new policy file. Can you help to share how it could be used, so I can help to update the installation guide.

Reply:
I pulled this in from the docs. We have a few dependencies here for querying the account. Again, I'm not sure if this is the minimum or if we can narrow it down.